integreat_cms/cms/models/events/event.py

```diff
@@ -149,10 +147,43 @@ def is_all_day(self):
         :return: Whether event takes place all day
         :rtype: bool
         """
-        return self.start_time == time.min and self.end_time == time.max.replace(
+        return self.start_local.time() == time.min and self.end_local.time() == time.max.replace(
             second=0, microsecond=0
```

Comment:
```suggestion
        return (
            self.start_local.time() == time.min
            and self.end_local.time() == time.max.replace(second=0, microsecond=0)
```
Our code formatting tool [black](https://github.com/digitalfabrik/integreat-cms/blob/develop/dev-tools/black.sh) wants to reformat these lines...

Reply:
Already resolved by [c79e687](https://github.com/digitalfabrik/integreat-cms/pull/1642/commits/c79e6870b2025d6d81e2ff24dcb2a496891c0b04)